README.md

```diff
@@ -919,13 +919,42 @@ extension your_class : RideRequestViewControllerDelegate {
 
 ## Example Apps
 
-Example apps can be found in the `examples` folder. To run build them, you need to use Carthage. (A quick overview of installing Carthage can be found in the **Getting Started** section.) From inside the `examples/Swift SDK` or `examples/Obj-C SDK` folder, run:
+Example apps can be found in the `examples` folder. To run build them, you can use Carthage or Cocoapods. 
+
+### Carthage
+(A quick overview of installing Carthage can be found in the **Getting Started** section.) From inside the `examples/Swift SDK` or `examples/Obj-C SDK` folder, run:
 
 ```
 carthage update --platform iOS
 ``` 
 This will build the required dependencies. Once you do that, open `Swift SDK.xcodeproj` or `Obj-C SDK.xcodeproj` in Xcode and run it.
 
+### CocoaPods
+You will have to remove Carthage dependencies. Remove Cartfile and Cartfile.resolved.
+
+From `examples/Swift SDK` or `examples/Obj-C SDK` folder, run:
+To integrate Uber Rides into your Xcode project, navigate to the directory `examples/Swift SDK` or `examples/Obj-C SDK` and create a new **Podfile** with `pod init`, then add `pod 'UberRides'` to the main loop. If you are using the Swift SDK, make sure to add the line `use_frameworks!`.
+
+```ruby
+use_frameworks!
+
+target 'Your Project Name' do
+pod 'UberRides'
+end
+```
+
+Then, run the following command to install the dependency:
+
+```bash
+$ pod install
+```
+
+For Objective-C projects, set the **Embedded Content Contains Swift Code** flag in your project to **Yes** (found under **Build Options** in the **Build Settings** tab).
+
+Open .xcworkspace and navigate to  **General** tab, scroll to **Embedded Binaries** select ObjectMapper.framework and click the `-` button, do the same for UberRides.framework. Now go to  **Build Settings** tab and scroll to **Search Paths**, click on *Framework Search Paths** and remove the line $(PROJECT_DIR)/Carthage/Build/iOS.
```

Comment:
\*Framework Search Paths\*\* is missing an asterisk, should be \*\*Framework Search Paths\*\*